internal/install/recipe_installer.go

```diff
@@ -148,22 +148,10 @@ func (i *RecipeInstaller) Install() error {
 
 	// Install integrations if necessary, continuing on failure with warnings.
 	if i.ShouldInstallIntegrations() {
-		for _, r := range recipes {
-			if !i.userAcceptsInstall(r) {
-				log.Infof("Skipping %s.", r.Name)
-				i.reportRecipeSkipped(execution.RecipeStatusEvent{
-					Recipe:     r,
-					EntityGUID: entityGUID,
-				})
-				continue
-			}
-
-			_, err := i.executeAndValidateWithProgress(m, &r)
-			if err != nil {
-				log.Warn(err)
-				log.Warn(i.failMessage(r.Name))
-			}
+		if err := i.installRecipesWithPrompts(m, recipes, entityGUID); err != nil {
```

Comment:
Extracted to method here since we hit the cyclomatic complexity limit for the `install` func.  Sorry it made this diff a little hard to rread.